nachomemes/__main__.py

```diff
@@ -1,6 +1,9 @@
 import sys
 from . import get_store
 
+# Render memes by default when calling:
+# python -m nachomemes <output_filename> <template_name> <text>
+# Example: python -m nachomemes bruh.png bruh bruh top / bottom
```

Comment:
Can we just slap this in the readme instead?

Reply:
this is a useful comment

Reply:
Which should be in the readme because it's instructions on how to use the bot... but ok.